Userland/Libraries/LibGfx/CMakeLists.txt

```diff
@@ -51,3 +51,6 @@ set(SOURCES
 
 serenity_lib(LibGfx gfx)
 target_link_libraries(LibGfx PRIVATE LibCompress LibCore LibCrypto LibTextCodec LibIPC)
+
+serenity_lib_static(LibGfxStatic gfx)
+target_link_libraries(LibGfxStatic PRIVATE LibCompressStatic LibCoreStatic LibCryptoStatic LibTextCodecStatic LibIPCStatic)
```

Comment:
Like discussed on discord, can we drop the mostly-static commit from the PR until we can see the utility of BuggieBox in the next set of changes and evaluate the cost of compiling all this code twice against the benefits to code size in the pre-boot environment at a later time?

Reply:
Done.